src/operators/operators.h

```diff
@@ -565,16 +565,20 @@ class local_Flux_Reconstruction_operator : public SumFactorizedOperators<dim,n_f
         const int nstate_input,
         const unsigned int max_degree_input,
         const unsigned int grid_degree_input,
-        const Parameters::AllParameters::Flux_Reconstruction FR_param_input);
+        const Parameters::AllParameters::Flux_Reconstruction FR_param_input,
+        const double FR_user_specified_correction_parameter_value_input);
```

Comment:
Since FR_user_specified_correction_parameter_value_input doesn't need have a value passed when using a c value from a formula/tabulated term (cDG, cHU, cPlus etc), the default could be set to 0 here in the constructor. Ie. change line 569 to:
 const double FR_user_specified_correction_parameter_value_input = 0.0);